### PR TITLE
Add exec inspect

### PR DESCRIPTION
**Pull request by tasuk**

### CODE REVIEW

**Comment by ubermuda** [comment]
Hey @tasuk, thanks for your contribution. Can you add unit tests and a bit of documentation please?

**Comment by tasuk** [comment]
@ubermuda: Done! The build fails for some unrelated reason.

**Comment by ubermuda** [comment]
Thank you! Could you move it to a test by itself though? Something like `testExecInspect`. This way we have a better way of what's going on in case of a problem.

**Comment by tasuk** [comment]
Separated the test!
(I just thought there'd be some unnecessary duplication, also the tests are hella slow.)

**Comment by ubermuda** [comment]
No problem that's a legitimate though, I just favour separation of tests over speed right now. They are indeed slow because they are not "unit tests" per se (they talk to a real docker daemon), and we sometimes need to blatantly `sleep()` to wait for a response, so yeah...

Anyway, thanks again for your contribution!

**Comment by tasuk** [comment]
Thanks for the quick merge! If you could do a new release, that'd be absolutely smashing :)

**Comment by ubermuda** [comment]
Sure thing! Just pushed v0.4.3

**Comment by tasuk** [comment]
:beers:

File: src/Docker/Manager/ContainerManager.php [hunk 1]
```diff
@@ -285,6 +285,28 @@ public function execstart($execid, callable $callback = null, $detach = false, $
     }
 
     /**
+     * Get details about an executive
+     *
+     * @param string $execid Identifier from exec()
+     *
+     * @throws \Docker\Exception\UnexpectedStatusCodeException
+     *
+     * @return stdClass
```
Comment (sroze) [inline src/Docker/Manager/ContainerManager.php]: That's inconsistent with the other methods that returns a `ResponseInterface` object btw :eyes:

Comment (ubermuda) [inline src/Docker/Manager/ContainerManager.php]: In the case of `exec()` you might need/want to interact with the output as a stream or whatever, which is not the case here, or did I miss something?

Comment (tasuk) [inline src/Docker/Manager/ContainerManager.php]: Just some methods return `ResponseInterface`. There's plenty of strings/arrays being returned too.

**Pull request merged**